pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiSchemaTest.java

```diff
@@ -185,6 +185,16 @@ private <T> void testSchemaInfoApi(Schema<T> schema,
 
     }
 
+    @Test
+    public void testCreateBytesSchema() {
+        // forbid admin api creating BYTES schema to be consistent with client side
+        try {
+            testSchemaInfoApi(Schema.BYTES, "schematest/test/test-BYTES");
```

Comment:
Please add `fail()` here. Otherwise, the test will get passed if the bytes schema is uploaded.